### PR TITLE
Point to SDK contributing guide

### PR DESCRIPTION
**Pull request by pq**
Redirect potential contributors to the linter's new home in the Dart SDK.

/cc @jacob314 @parlough @bwilkerson

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,87 +1,15 @@
 Contributing to the Linter
 ==========================
 
-[![Build Status](https://github.com/dart-lang/linter/workflows/linter/badge.svg)](https://github.com/dart-lang/linter/actions)
-![GitHub contributors](https://img.shields.io/github/contributors/dart-lang/linter.svg)
+## [Repository has moved][]
```
Comment (bwilkerson) [inline CONTRIBUTING.md:4]: I don't see a target for this link. I'm guessing that you didn't intend for it to be a link.

**Review by bwilkerson — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (pq) [reply CONTRIBUTING.md:4]: Quite right. I just copied what @parlough did for the README but perhaps that's a mistake too?

Comment (pq) [reply CONTRIBUTING.md]: Fixed.

**Review by parlough — APPROVED** [review]
Good idea to update this too!

**Pull request merged**